frontend/src/common/models/formik/initialValuesForgotPassword.ts

```diff
@@ -0,0 +1,7 @@
+import TForgotPassword from '../auth/TForgotPassword';
+
+const initialValuesForgotPassword: TForgotPassword = {
+  email: ''
+};
+
+export default initialValuesForgotPassword;
```

Comment:
No need to add this to the models folder, it's not a model. We write initial values directly in components. Same for initialValuesResetPassword

Reply:
Solved